src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java

```diff
@@ -66,6 +66,132 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class FirstClass extends Exception {
+ *   private int code; // OK, class name doesn't match with default pattern
+ *
+ *   public FirstClass() {
+ *     code = 1;
+ *   }
+ * }
+ *
+ * class MyException extends Exception {
+ *   private int code; // violation, The field 'code' must be declared final
+ *
+ *   public MyException() {
+ *     code = 2;
+ *   }
+ * }
+ *
+ * class MyThrowable extends Throwable {
+ *    final int code; // OK
+ *    String message; // violation, The field 'message' must be declared final
+ *
+ *    public MyThrowable(int code, String message) {
+ *      this.code = code;
+ *      this.message = message;
+ *    }
+ * }
+ *
+ * class BadException extends java.lang.Exception {
+ *   int code; // violation, The field 'code' must be declared final
+ *
+ *   public BadException(int code) {
+ *     this.code = code;
+ *   }
+ * }
+ * </pre>
+ * <p>
+ * To configure the check so that it only checks for class names
+ * that match pattern:
```

Comment:
> only 

this word confused me in previos review item. Lets remove  it.


it checks for class name that ends with 'Exception':

Reply:
Done